src/Mod/PartDesign/Gui/ReferenceSelection.cpp

```diff
@@ -104,7 +116,7 @@ bool ReferenceSelection::allow(App::Document* pDoc, App::DocumentObject* pObj, c
 
 PartDesign::Body* ReferenceSelection::getBody() const
 {
-    PartDesign::Body *body;
+    PartDesign::Body *body = nullptr;

```

Comment:
This assignment isn't needed, it is set in both the if and else block.
If you want to you can use the ternary operator to reduce the function to a single line. But that's totally optional.